.github/workflows/build.yml

```diff
@@ -0,0 +1,77 @@
+name: build
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - 'master'
+      - 'v2'
+    tags:
+      - 'v*'
+  pull_request:
+    branches:
+      - 'master'
+      - 'v2'
+
+jobs:
+  validate:
+    runs-on: ubuntu-latest
+    steps:
+      -
+        name: Checkout
+        uses: actions/checkout@v2
```

Comment:
Was there a reason the checkout action was needed here?

Reply:
The bake action can't resolve git context like build-push action does atm. I think we could impl a remote context detection for some CI providers based on env vars like codecov does with their [bash uploader script](https://github.com/codecov/codecov-bash/blob/master/codecov) on buildx repo:

```hcl
target "base" {
    inherits = ["meta-helper"]
    providers = ["github", "travis"]
    context = "base"
    target = "base"
}
```

WDYT?

Reply:
Or it could just be the default if there is no local bake file and there is github actions context.